src/cascadia/TerminalSettingsModel/FontConfig.idl

```diff
@@ -16,5 +16,15 @@ namespace Microsoft.Terminal.Settings.Model
         INHERITABLE_FONT_SETTING(String, FontFace);
         INHERITABLE_FONT_SETTING(Int32, FontSize);
         INHERITABLE_FONT_SETTING(Windows.UI.Text.FontWeight, FontWeight);
+
+        Windows.Foundation.Collections.IMap<String, UInt32> FontFeatures;
+        Boolean HasFontFeatures { get; };
+        void ClearFontFeatures();
+        Microsoft.Terminal.Settings.Model.FontConfig FontFeaturesOverrideSource { get; };
+
+        Windows.Foundation.Collections.IMap<String, Int32> FontAxes;
+        Boolean HasFontAxes { get; };
+        void ClearFontAxes();
+        Microsoft.Terminal.Settings.Model.FontConfig FontAxesOverrideSource { get; };

```

Comment:
The `INHERITABLE_FONT_SETTING` macro doesn't like `Windows.Foundation.Collections.IMap<String, UInt32>` (possibly because of the comma) so I'm declaring these manually for now

Reply:
(this may make sense as a comment in the code itself)